4-toolchain/Dockerfile.rhel7

```diff
@@ -52,12 +51,12 @@ WORKDIR ${HOME}
 
 # Use entrypoint so path is correctly adjusted already at the time the command
 # is searching, so something like docker run IMG gcc runs binary from SCL.
-COPY contrib/bin/container-entrypoint /usr/bin/container-entrypoint
+COPY root/opt/app-root/bin/container-entrypoint /usr/bin/container-entrypoint
 
 # Install the usage script with base image usage informations
-COPY contrib/bin/usage /usr/local/bin/usage
+COPY root/opt/app-root/bin/usage /usr/local/bin/usage
 
-COPY contrib/etc/scl_enable /opt/app-root/etc/scl_enable
+COPY root/opt/app-root/etc/scl_enable /opt/app-root/etc/scl_enable
```

Comment:
Maybe we needs. Or correct permissions have to be set.

So change `chmod u+x /opt/app-root/bin/usage` to `chmod a+rx /usr/bin/usage /usr/bin/container-entrypoint` (or `/opt/app-root/bin` if these files should be there) ?